packages/react/src/components/ParticipantLoop.tsx

```diff
@@ -48,7 +48,7 @@ type ParticipantLoopProps = {
 export const ParticipantLoop = ({
   filter,
   filterDependencies,
-  updateOnlyOn,
+  updateOnlyOn = [],
```

Comment:
This reduces the updates to the three base events. Probably not what we want.

Reply:
hm, as long as there's no filter supplied, I would say it _is_ actually what we want